release-notes/aws/10.1.0.md

```diff
@@ -0,0 +1,40 @@
+## :zap:  Giant Swarm Release 10.1.0 for AWS is now active for you! :zap:
+
+We are pleased to announce that this new branch (10.1.0) contains the much anticipated Node Pools release and will contain all
+other releases thereafter such as High Availability of Masters and Spot Instances which are due to be released next year. 
+
+We therefore kindly remind you that from this release and for a period of six months onwards, 
+to give you a chance to move your clusters, we will be supporting both this new branch and the 'Legacy' 9.0.0 branch which we will
+continue to maintain during this period. 
+
+In the meantime, we hope you enjoy using Node Pools of which the documentation can be found here: https://docs.giantswarm.io/basics/nodepools/ 
+including the other release changes which are listed below: 
+
+### Kubernetes v1.15.5
+- Updated from v1.14.6 - [changelog](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.15.md#kubernetes-v115-release-notes)
+- Includes a fix for CVE-2019-11253 related to json/yaml decoding where large or malformed documents could consume excessive server resources. Request bodies for normal API requests (create/delete/update/patch operations of regular resources) are now limited to 3MB.
+
+### Calico v3.9.1
+- Updated from v3.8.2 - [changelog](https://docs.projectcalico.org/v3.9/release-notes/)
+
+### CoreOS Container Linux v2191.5.0
+- Updated from v2135.4.0 - [changelog](https://coreos.com/releases/#2191.5.0)
+
+### etcd v3.3.15
+- Updated from v3.3.13 - [changelog](https://github.com/etcd-io/etcd/blob/master/CHANGELOG-3.3.md#v3315-2019-08-19)
+
+### kube-state-metrics v1.8.0 ([GS v0.6.0](https://github.com/giantswarm/kube-state-metrics-app/blob/master/CHANGELOG.md#v060))
+- Updated from upstream `kube-state-metrics` v1.7.2 - [changelog](https://github.com/kubernetes/kube-state-metrics/blob/master/CHANGELOG.md#v180--2019-10-)
+
+### nginx-ingress-controller v0.26.1 ([GS v1.0.0](https://github.com/giantswarm/kubernetes-nginx-ingress-controller/blob/master/CHANGELOG.md#100))
+- Updated from upstream `ingress-nginx` v0.25.1 - [changelog](https://github.com/kubernetes/ingress-nginx/blob/master/Changelog.md#0261)
+- **Note** Includes the following breaking changes
+  - The variable `$the_real_ip` was removed from template and default `log_format`.
+  - The default value of configmap setting `proxy-add-original-uri-header` is now `false`.
+  - When the setting `proxy-add-original-uri-header` is `true`, the ingress controller adds a new header `X-Original-Uri` with the value of NGINX variable `$request_uri`. In most of the cases this is not an issue but with request with long URLs it could lead to unexpected errors in the application defined in the `Ingress` `serviceName`, like issue https://github.com/kubernetes/ingress-nginx/issues/4593.
```

Comment:
this release does NOT include any NGINX ingress controller, it is fully optional from this release on. @cokiengchiara we should write a big note in the top about this. @J-K-C you ok with us adding that note?

Reply:
yes please go ahead!!!

Reply:
whilst you're here @puja108  - please could you check that all the release details which I copied from release 9.0.0 are correct please..

Reply:
additionally there is 3 added components here namely: external-dns, cert-manager, kiam

Reply:
@MarcelMue is this something you can help me add please?

Reply:
From the discussion above it seems that Chiara knows what to add here and should provide some text to include.

Reply:
Can we replace this with:

```
### external-dns v0.5.11 ([GS v1.0.0](https://github.com/giantswarm/external-dns-app/blob/master/CHANGELOG.md#v100))
- Added as a default app [Upstream Changelog](https://github.com/kubernetes-sigs/external-dns/releases/tag/v0.5.11)

### cert-manager v0.9.0 ([GS v1.0.1](https://github.com/giantswarm/cert-manager-app/blob/master/CHANGELOG.md#v101))
- Added as a default app [Upstream Changelog](https://github.com/jetstack/cert-manager/releases/tag/v0.9.0)

### kiam v3.4 ([GS v1.0.0](https://github.com/giantswarm/kiam-app/blob/master/CHANGELOG.md#v100))
- Added as a default app [Upstream Changelog](https://github.com/uswitch/kiam/releases/tag/v3.4)
```